src/Main.kt

```diff
@@ -1,14 +1,12 @@
-//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
-// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 fun main() {
     val name = "Kotlin"
-    //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
-    // to see how IntelliJ IDEA suggests fixing it.
     println("Hello, " + name + "!")
 
+    extracted()
+}
+
+private fun extracted() {
```

Comment:
Could you please explain what this function does and rename it according your explanation?